### DISCUSSION TITLE
Target #53 - overlayed color?

### DISCUSSION THREAD

**Discussion by dxvalchovski** [cssbattle/forum#225] [Q&A]
Hey there,

Been trying to improve my high score on target #53, however, I was wondering if there may be an RGBA format color that can achieve the overlay of the circle and the rectangle?

My minified snippet so far can be found below:
`*{background:#19191a}*>*{width:75;height:125;margin:125;background:#f9e492}*>:before{content:'';position:absolute;top:50;border-radius:50%;border:75px solid#4f77ff;border-bottom-color:#9ad5ff;transform:rotate(45deg`

 Wondering what's the secret sauce behind the 130-150 char solutions -- any tips appreciated!

**Comment by SelenIT** [comment]
RGBA color (in 4-hex-digit form) is indeed part of shortest solutions. But there are other ways to create circular segments than rotating border parts. Also, like "the best code is no code", maybe sometimes the best overlay is no overlay?..

**Reply by verberg** [reply]
Am I right?  A 4-hex-digit form is the short hand from #aabbccdd -->#abcd ?

**Reply by SelenIT** [reply]
Yes, exactly!
